### PR TITLE
docs: Add initial LMAT instructions

### LINKED ISSUE #3: Docs

### PR DESCRIPTION
**Pull request by Callisto13**
Close #3 

Requires #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,16 +9,226 @@ and run: mdtoc -inplace README.md
...
+  - [Locally (option 1)](#locally-option-1)
+    - [Tunables](#tunables)
+  - [Locally (option 2)](#locally-option-2)
+  - [Locally (option 3)](#locally-option-3)
   - [In CI](#in-ci)
 <!-- /toc -->
 
 ## What they test
 
+The LMATS are the highest level suite for the Liquid Metal project. Thus they
```
Comment (richardcase) [inline README.md]:
```suggestion
The LMATS are the highest level test suite for the Liquid Metal project. Thus they
```

File: README.md [hunk 2]
```diff
@@ -9,16 +9,226 @@ and run: mdtoc -inplace README.md
...
+The test code itself for now lives in [CAPMVM][capmvm-e2e].
+
+There are 2 main parts to this repo:
+- [`terraform/`][tf] which contains manifests for provisioning bare-metal infrastructure
+   and configuring flintlock.
+- [`cmd/`][tool] which triggers the execution of the tests.
+
+The sequence of events for a full run is:
+- Terraform section...
+  - Check capacity of Equinix for requested device types and elect metro with
```
Comment (richardcase) [inline README.md]:
```suggestion
  - Check capacity of Equinix for requested device types and select metro with
```

File: README.md [hunk 3]
```diff
@@ -9,16 +9,226 @@ and run: mdtoc -inplace README.md
...
+  - Prepare the "management" host to run CAPI
+- Test runner section (over SSH to the management host)...
+  - Prepare configuration based on the output of the Terraform step and any
+    action inputs
+  - Clone CAPMVM on the management host
+  - Change into the directory and run the e2e tests
+- E2E section (streamed over SSH from the management host)...
+  - Create a kind cluster
+  - Initialise the cluster with required CAPI controllers
+  - Generate a template for the CAPMVM workload
```
Comment (richardcase) [inline README.md]:
```suggestion
  - Generate a template for the CAPMVM workload cluster
```

File: README.md [hunk 4]
```diff
@@ -9,16 +9,226 @@ and run: mdtoc -inplace README.md
...
+  - Apply the workload to the kind cluster
```
Comment (richardcase) [inline README.md]:
```suggestion
  - Apply the workload cluster yaml to the kind cluster
```

File: README.md [hunk 5]
```diff
@@ -9,16 +9,226 @@ and run: mdtoc -inplace README.md
...
+
+This process is quite lengthy, you are looking at 10-20 mins. The test section
+alone can take up to 5 mins to run (I am working on making that faster).
+
+To work in steps, or to run the tests several times with the same infrastructure,
+you can call the individual targets:
+
+```bash
+make tf-up
+make e2e # add any flags here as E2E_ARGS="--foo bar" see 'Tunables' below for more
```
Comment (richardcase) [inline README.md:111]: Worth adding in the comment that this step is the one that can be run multiple times?

File: cmd/README.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+
+This tool will go away when I have figured out some more networking for the infra.
+
+## What and why
+
+There are 2 reasons it exists:
+1. To save time on networking complexity during my initial stab at these tests,
+  I chose not to set it up so that the CAPI management cluster could be run
+  from outside the Equinix infra network.
+  _Technically_ then can be since the flintlock servers are bound to a public
```
Comment (richardcase) [inline cmd/README.md]: Not sure about the then

```suggestion
  _Technically_ it could be since the flintlock servers are bound to a public
```

Comment (Callisto13) [reply cmd/README.md]: think this might have been a `they` 🤔

File: README.md (hunk 5 above)
Comment (Callisto13) [reply README.md:111]: yeh probs worth making it very explicit

**Review by richardcase — APPROVED** [review]

**Comment by Callisto13** [comment]
oh forgot about this

**Pull request merged**